data-prepper-plugins/geoip-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/databasedownload/HttpDBDownloadService.java

```diff
@@ -42,13 +41,13 @@ public HttpDBDownloadService(String prefixDir) {
      * Initialisation of Download through Url
      * @param urlList urlList
      */
-    public void initiateDownload(List<DatabasePathURLConfig> urlList) throws Exception {
+    public void initiateDownload(List<String> urlList) {
```

Comment:
This entire package (`databasedownload`) could probably be put under the `.extension` package as well - `.extension.databasedownload`.

Again, this could be a follow-on PR.